vulnerability-report.md

```diff
@@ -0,0 +1,75 @@
+##Vulnerability Report
+
+###Reviewer: Wenjing Qiang
+
+###Date: May 11, 2016
+
+
+###Injection
+
+Exploiting: Tying singel sql in uploadfile field, programeError will show.
```

Comment:
Not supposed to grade on spelling but this is borderline illegible. Also, into what page were you typing SQL?